1209/EAEA/index.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: pid
+title: Pinscape Controller
```

Comment:
FYI, as-is this will cause it to show up in the list as "Pinscape Pinscape Controller". Delete "Pinscape" here if you don't want that to be the case, or let me know if you want to go ahead and merge it as-is.